src/components/Tooltip.tsx

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+import Tippy from '@tippyjs/react';
+
+export const Tooltip = styled(Tippy)`
```

Comment:
To keep consistent with the rest of the code, and since this component does not accept any params, could you add it directly to AccountModal.tsx, where it is being used, at the end of the file?

Reply:
Moved the style tippy component to the end of the AccountModal file and deleted the Tooltip.tsx file